docs/guide/Configs/Toolchains.md

```diff
@@ -19,10 +19,10 @@ BenchmarkDotNet allows you to take full advantage of that. With single config, w
 If you specify `Runtime` in explicit way, we just choose the right toolchain for you.
 
 ```cs
-[ClrJob, MonoJob, CoreJob]
+[ClrJob, MonoJob, CoreJob, CoreRtJob]
 public class Algo_Md5VsSha256
 {
-    // the benchmarks are going to be executed for classic .NET, Mono (default path) and .NET Core
+    // the benchmarks are going to be executed for classic .NET, Mono (default path), .NET Core and CoreRT (latest version)
```

Comment:
@adamsitnik could you please specify the supported versions instead of `lastest version`?

Reply:
it's a little bit more tricky with CoreRT when it comes to versioning. Currently it's only `1.0.0-alpha-xx-yy` 

What I meant here by latest version is that if the user does not specify the version in explicit way, BDN will restore ` 1.0.0-alpha-*` of `https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.DotNet.ILCompiler` which effectuvely will use latest version of CoreRT.

I am going to add a CoreRT section to docs/toolchain an explain all the details there.